### PR TITLE
Adds display options tab

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
displaytab

### CODE REVIEW

File: www/js/desktop_scripts.js [hunk 1]
```diff
@@ -148,22 +148,36 @@ Array.from($titleButtons).forEach((el) => {
...
   document.querySelector('.tab-content.active').classList.remove('active');
   tabContent.classList.add('active');
 }
 
-const allTabs = document.getElementsByClassName('nav-header-tab');
-
 Array.prototype.forEach.call(allTabs, (element => {
-  element.addEventListener('click', showTabContent);
+  element.addEventListener('click', () => {
+    const clickedTab = event.currentTarget.id;
```
Comment (tsingh777) [inline www/js/desktop_scripts.js]: should `event` not be defined first? Seems to be working but style wise doesn't seem consistent with other event listeners. 

```
event => {
    const clickedTab = event.currentTarget.id;
    showTabContent(clickedTab);
    ...
```

File: www/src/scss/_header.scss [hunk 2]
```diff
@@ -105,3 +118,9 @@
     display: block;
   }
 }
+
+.presenter-view.scale-viewer .more-tabs {
+  @media screen and (max-width: 850px) {
```
[the remark this answers is not in the document]
Comment (inderpreetsingh) [reply www/src/scss/_header.scss]: We can do this specific improvement later, @Gauravjeetsingh can we please complete rest of stuff and get this PR merged?

File: www/index.html [hunk 3]
```diff
@@ -87,9 +87,16 @@
             <span class="nav-header-tab nav-header-text" id="insert-tab">
               <i class="fa fa-desktop"></i> Insert
             </span>
-            <span class="nav-header-tab nav-header-text" id="other-tab">
-                <i class="fa fa-ellipsis-h"></i> Other
-              </span>
+            <div class="list">
```
Comment (tsingh777) [inline www/index.html]: On Mac it doesn't really look nice.
bug

It also doesn't behave the same as the video in the PR. takes 2 clicks to go to the selection when changing the selection.

@saintsoldierx should comment but my suggestion would be not use a select tag for the extra tags.

**Comment by inderpreetsingh** [comment]
@Gauravjeetsingh can you please take a look at this?

**Comment by Gauravjeetsingh** [comment]
Switched to custom dropdown instead of select

aug 21 2018 11_52 pm

**Comment by tsingh777** [comment]
Agree with @inderpreetsingh 
> Actually, if there's space on the right hand side, I think there should be no select at all. It's only when there's no space on right hand side, and the box is small, only then we should have that dropdown. That will also show selectbox at only edge cases (small screens) and we might tolerate that.

We can follow up in separate PR.

**Pull request merged**